### PR TITLE
dev-lang/erlang: version bump to 22.1.1

### PR DESCRIPTION
**Pull request by arkamar**
I made also changes in service scripts to run Erlang Port Mapper Daemon (epmd) as an `epmd` user rather than `root` in addition to a version bump. The new `epmd` UID and GID is set to 334.

### CODE REVIEW

**Comment by arkamar** [comment]
I changed the UID/GID to 335 in reaction to https://archives.gentoo.org/gentoo-dev/message/03252c9d9eb4c34f7dfa569b484ebcab

File: dev-lang/erlang/erlang-22.1.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+	doc? ( http://erlang.org/download/otp_doc_html_${UPSTREAM_V}.tar.gz -> ${PN}_doc_html_${UPSTREAM_V}.tar.gz )"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~x64-solaris"
+IUSE="doc emacs +hipe java +kpoll libressl odbc sctp ssl systemd tk wxwidgets"
+
+RDEPEND="
+	acct-user/epmd
+	acct-group/epmd
```
Comment (juippis) [inline dev-lang/erlang/erlang-22.1.1.ebuild]: You could sort group before user, but I understand if you want to keep it this way too. Rest looks good!

File: dev-lang/erlang/erlang-22.1.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+	"${FILESDIR}/18.2.1-wx3.0.patch"
+	"${FILESDIR}/${PN}-22.0-dont-ignore-LDFLAGS.patch"
+)
+
+SITEFILE=50"${PN}"-gentoo.el
+
+src_prepare() {
+	default
+
+	./otp_build autoconf
```
[the remark this answers is not in the document]
Comment (arkamar) [reply dev-lang/erlang/erlang-22.1.1.ebuild]: Thanks, it's fixed.

File: dev-lang/erlang/erlang-22.1.1.ebuild (hunk 1 above)
Comment (arkamar) [reply dev-lang/erlang/erlang-22.1.1.ebuild]: I swapped them.

**Comment by juippis** [comment]
Thanks, works great!

**Pull request closed without merging**